src/components/SelectAvatar/styles.js

```diff
@@ -1,12 +1,21 @@
 import { StyleSheet } from 'react-native';
 
 const styles = StyleSheet.create({
-  image: {
-    width: 80,
-    height: 80,
-    borderRadius: 40,
-    alignSelf: 'center',
-    marginVertical: 10,
+  container: {
+    alignItems: 'center',
+    marginRight: 80,
+    marginTop: 80,
```

Comment:
The profile avatar should be centered on all screen dimensions, I think this is not going to work in all cases

Reply:
The image is center aligned, but moved to the left to center the blue circle. So, it works for different screen widths. I mean, these values refer to the image dimensions, that are always the same.